### PR TITLE
[dev-2.6] Upgrade rke2 versions

### PR DESCRIPTION
**Pull request by dweomer**
- https://github.com/rancher/rke2/issues/949 (1.18.19)
- https://github.com/rancher/rke2/issues/950 (1.19.11)
- https://github.com/rancher/rke2/issues/951 (1.20.7)

### CODE REVIEW

**Comment by davidnuzik** [comment]
Version bumps LGTM, would like @kinarashah to review as I don't grok the 2.6 changes in KDM and ensure what's in the yaml is good to go.

File: channels-rke2.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  - version: v1.18.17+rke2r1
+  - version: v1.18.19+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.19.9+rke2r1
+  - version: v1.19.11+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.20.5+rke2r1
+  - version: v1.20.7+rke2r1
```
Comment (davidnuzik) [inline channels-rke2.yaml]: @dweomer can you bump to v1.20.7+rke2r2 at your convenience this week?

**Review by davidnuzik — CHANGES_REQUESTED** [review]
bump to v1.20.7+rke2r2

**Review by kinarashah — APPROVED** [review]

**Review by davidnuzik — APPROVED** [review]

**Comment by davidnuzik** [comment]
@kinarashah think we can merge this? Or are we not comfortable yet? e.g. I'm thinking about @ibuildthecloud 's changes in 2.6 and the new entries in channels yaml like the [charts](https://github.com/rancher/kontainer-driver-metadata/pull/559/files#diff-a0456795a97052d90c30065cde4425e798e52b4a119590fc8a7e66416ec5ce61R11), serverArgs, etc, etc. I don't grok the changes in 2.6 dev branch, and so I don't necessarily feel comfortable merging this in...

**Comment by kinarashah** [comment]
@davidnuzik Based on what I understand, those entries don't affect v1.18 and v1.19. 
For v1.20, it was already in use so bumping k8s version should be good because what worked for `v1.20.5` will work for `v1.20.7`. But I could be wrong because I don't fully understand those args either. 

Anyway, it's a dev branch and we don't have a release-v2.6 planned anytime soon, so I'd say let's merge and get it to test so we'll know what needs to be fixed. Worst case, we revert in case of blocker issues. 

Thoughts?

**Comment by davidnuzik** [comment]
@kinarashah gotcha. And yes, very true no 2.6 releases any time very soon I believe. Ok let's merge this.

**Pull request merged**